internals/scripts/setup.js

```diff
@@ -68,6 +52,33 @@ function deleteFileInCurrentDir(file, callback) {
 /**
  * Installs dependencies
  */
-function installDeps(callback) {
-  exec('npm install', addCheckMark.bind(null, callback));
+function installDeps() {
+  exec('yarn --version', function (err, stdout, stderr) {
+    if (parseFloat(stdout) < 0.15 || err || process.env.USE_YARN === 'false') {
```

Comment:
What is this for: `parseFloat(stdout) < 0.15`? Can we add a comment above stating why we don't want versions lower than 0.15?